apps/dashboard/src/app/account/account-home/account-home.component.html

```diff
@@ -122,7 +122,20 @@
     </div>
 
     <div class="right-column">
-      <div class="container kb-state">
+      <div
+        class="container plan"
+        routerLink="../billing">
```

Comment:
I remember there's a feature flag to enable/disable the billing page, maybe we should use it here.